vllm/__init__.py

```diff
@@ -17,6 +17,43 @@
 
 from .version import __version__, __version_tuple__
 
+
+def configure_as_vllm_process():
+
+    # some common environment variables
+    # for all processes created by vllm
+
+    import os
+
+    import torch
+
+    # see https://github.com/NVIDIA/nccl/issues/1234
+    os.environ['NCCL_CUMEM_ENABLE'] = '0'
+
+    # see https://github.com/vllm-project/vllm/issues/10480
+    os.environ['TORCHINDUCTOR_COMPILE_THREADS'] = '1'
+    # see https://github.com/vllm-project/vllm/issues/10619
+    torch._inductor.config.compile_threads = 1
+
+    from vllm.platforms import current_platform
+
+    if current_platform.is_xpu():
+        # see https://github.com/pytorch/pytorch/blob/43c5f59/torch/_dynamo/config.py#L158
+        torch._dynamo.config.disable = True
+    if current_platform.is_hpu():
```

Comment:
Just to be clear
```suggestion
    elif current_platform.is_hpu():
```

Reply:
fixed in [98b417d](https://github.com/vllm-project/vllm/pull/12084/commits/98b417d3fc4c5a1fa5af18e7d375592474f65dfa)